ad_source/serializers.py

```diff
@@ -65,7 +65,7 @@ def validate(self, attrs):
             if og.response.status_code != status.HTTP_200_OK:
                 message = f'Website {website_link} responded with status code: {og.response.status_code}'
                 L.warning(message)
-                raise serializers.ValidationError({'website_link': f'{message}. Has to be 200'})
+                # raise serializers.ValidationError({'website_link': f'{message}. Has to be 200'})
```

Comment:
because it was raising that error way too often which would prevent the campaign from being created, it needs to be replaced with a warning or something